dag/main.yml

```diff
@@ -337,6 +337,14 @@ steps:
   data://grapher/happiness/2023-03-20/happiness:
   - data://garden/happiness/2023-03-20/happiness
 
+  # WB income groups
+  data://meadow/wb/2023-04-30/wb_income:
```

Comment:
Now that you are here... I have never liked the dataset short name "wb_income". It would make way more sense to call this dataset something "income_groups", wouldn't it? Up to you, if it's too much hassle leave it.